torch/csrc/jit/tensorexpr/llvm_codegen.cpp

```diff
@@ -518,6 +519,13 @@ void LLVMCodeGenImpl::emitKernel(
   if (llvm::verifyFunction(*fn_, &llvm::outs())) {
     throw std::runtime_error("Function verification failed");
   }
+
+  // print graph debug info.
+  std::string fnstr;
```

Comment:
It's not ideal that we're dumping the llvm into a string even if graph_debug isn't on, slowing down all compilation

Reply:
Agree. This was fixed in a follow-up issue (issue #48771, commit e99d7df).